_component_design/tooltip.md

```diff
@@ -267,16 +244,23 @@ Use this variation when a tooltip provides access to a singular external webpage
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Large tooltips
+#### Click interaction
+
+Use click-enabled tooltips for smaller breakpoints and mobile designs. Tooltips can have both hover and click interactions on the same tooltip across breakpoints.
```

Comment:
I thought you weren't supposed to use tooltips when users are working on a mobile device...